packages/yoshi-common/src/dev-environment-logger.ts

```diff
@@ -147,6 +154,14 @@ export type DevEnvironmentLogger = (opts: {
   suffix: string;
 }) => void;
 
+function logUsageInstructions(state: State) {
+  if (state.AppServer) {
+    console.log(chalk.bold('Usage'));
+    console.log(` › Press ${chalk.bold('r')} to hard restart the app server`);
```

Comment:
```suggestion
    console.log(` › Press ${chalk.bold('r')} to restart the app server`);
```

I know that technically `hard restart` may be more correct, but I guess people are going to see this quite often and I think that `restart` will do the trick